source/ranges.tex

```diff
@@ -866,6 +864,11 @@
 \end{itemdecl}
 
 \begin{itemdescr}
+\pnum
+\tcode{T} models \libconcept{View} only if the execution time
+for the the initializations and assignments required by \libconcept{Copyable<T>}
```

Comment:
duplicate "the"
Also, we should go singular here to avoid the question whether we mean each or all (together).
"if the execution time for each initialization and assignment required by ... is independent of the number of elements in T"
(If it's O(n^2), it's also "not proportional"; we want to say something stronger.)

Reply:
Sure. I applied the use of "complexity of ... constant time" and reverted the first paragraph of the description.

Reply:
Thanks.

Reply:
Do you think the previous sibling clause, [range.sized], also needs to be modified? That's where I took the use of O(1) from:
https://github.com/cplusplus/draft/blob/3d9f80e990daf459a85cff539bd2ce64c0986886/source/ranges.tex#L797